controller/src/test_controller/action.rs

```diff
@@ -124,12 +128,25 @@ async fn resource_readiness(t: &TestInterface) -> Result<Resources> {
     let resources_names = &t.test().spec.resources;
     for resource_name in resources_names {
         let result = resource_client.get(resource_name).await;
-        if result.is_status_code(StatusCode::NOT_FOUND) {
-            return Ok(Resources::Error(format!(
-                "Resource '{}' not found",
-                resource_name
-            )));
+
+        let mut retry_attempt = 1;
+        while retry_attempt <= MAX_RETRIES {
```

Comment:
```suggestion
        let mut retry_attempt = 0;
        while retry_attempt < MAX_RETRIES {
```
Super nit

Reply:
The reason I start attempts with 1 is inside `while` logic we have an if condition to error out if we try out all the attempts.`if retry_attempt == MAX_RETRIES {` . If I start it with 0, I need modify it to `if retry_attempt == MAX_RETRIES - 1{`, which is less readable.